modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -1206,6 +1212,27 @@ private void _validateRequiredTypeSettingsKeys(
 		}
 	}
 
+	private void _validateTypeSettingsBoolean(
+			ClientExtension clientExtension, boolean required,
+			String typeSettingsKey)
+		throws GradleException {
+
+		Object typeSettingsValue = clientExtension.typeSettings.get(
+			typeSettingsKey);
+
+		if (((typeSettingsValue == null) && !required) ||
+			(typeSettingsValue instanceof Boolean)) {
+
+			return;
+		}
+
+		throw new GradleException(
+			String.format(
+				"Client extension %s must have a boolean value for the " +
+					"property %s",
+				clientExtension.id, StringUtil.quote(typeSettingsKey)));
```

Comment:
What is the purpose of the `required` parameter here? That should be covered by `_validateRequiredTypeSettingsKeys` already. This one's just asserting a type, right?

Reply:
You are right, thanks for the heads up